tests/negative/unicode_expect_failure_test.toit

```diff
@@ -0,0 +1,13 @@
+import expect show *
```

Comment:
Add a copyright.

Also add a comment, what this test is for. (refer to the issue, and a small explanation).